src/core.rs

```diff
@@ -3440,9 +3482,10 @@ impl Select {
         lines.push("SELECT *,".to_string());
         // WARN: The _total count should probably be optional.
         lines.push("  COUNT(1) OVER() AS _total, ".to_string());
+        // WARN: Postgres only!
         lines.push(format!(
             r#"(SELECT MAX(change_id) FROM history
-                  WHERE "table" = ?
+                  WHERE "table" = $1
```

Comment:
This is wrong and lazy, but we don't have immediate access to the `DbKind` here.